python/cudf_polars/cudf_polars/experimental/rapidsmpf/io.py

```diff
@@ -401,6 +397,71 @@ async def _producer(ch_out: Channel) -> None:
         await asyncio.gather(*tasks)
 
 
+def make_rapidsmpf_read_parquet_node(
```

Comment:
@wence- - Anything fundamentally _wrong_ here? I'm hitting a segfault any time the `read_parquet` node is actually used.